dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestratorTest.kt

```diff
@@ -415,6 +438,29 @@ internal class BatchFileOrchestratorTest {
 
     // endregion
 
+    // region getNewWritableFile
+
+    @Test
+    fun `𝕄 return new File 𝕎 getWritableFile() {forceNewFile=true}`() {
+        // Given
+        assumeTrue(fakeRootDir.listFiles().isNullOrEmpty())
+
+        // When
+        val start = System.currentTimeMillis()
+        val result = testedOrchestrator.getWritableFile(forceNewFile = true)
```

Comment:
I think result with `forceNewFile = false` will be the same, because it is the fresh state of the orchestrator and it will always return the new file.

What we need to do is to call `getWritableFile` twice (one is default call, another with `forceNewFile = true`) and check that second result is not the same as first and was created after the first call.